appgate/syncer/git.py

```diff
@@ -138,3 +174,36 @@ def get_git_repository(ctx: GitOperatorContext) -> GitRepo:
         return github_repo(ctx, GIT_DUMP_DIR)
     else:
         raise Exception(f"Unknown git vendor type {ctx.git_vendor}")
+
+
+@functools.cache
+def entity_path(repository_path: Path, kind: str) -> Path:
+    return repository_path / kind.lower()
+
+
+@attrs()
```

Comment:
Time to use the new attrs API I think see https://www.attrs.org/en/stable/examples.html

Reply:
Can we do that once the feature is merged? I think it's making things easier

Reply:
Sure